donut.html

```diff
@@ -4,6 +4,10 @@
   <meta charset="UTF-8">
   <title>Donut Shop</title>
   <link rel="stylesheet" href="donut.css">
+<<<<<<< HEAD
+  <link href='http://fonts.googleapis.com/css?family=Eater' rel='stylesheet' type='text/css'>
+=======
+>>>>>>> 914f29507db48ce2da2bb92b6197a6be832c4cca
```

Comment:
Looks like you have a merge conflict here. Did you get this resolved in a later commit/push?